snow/networking/router/main_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package router
+
+import (
+	"testing"
+
+	"go.uber.org/goleak"
+)
+
+// TestMain uses goleak to verify tests in this package do not leak unexpected
+// goroutines.
```

Comment:
nit: if the code is self-documenting I usually don't put a comment on it, i would prefer to just delete this comment

Reply:
Dropped all of them